config/peerpods/podvm/lib.sh

```diff
@@ -110,9 +119,14 @@ install_binary_packages() {
             fi
 
             echo "Marking  ${install_dir}/${package_name} executable"
-            # yq extracted file name is yq_linux_amd64. Rename it
-            [[ "${package_name}" == "yq" ]] &&
-                mv "${install_dir}/yq_linux_amd64" "${install_dir}/yq"
+            # yq extracted file name is yq_linux_${ARCH}. Rename it
+            if [[ "${package_name}" == "yq" ]]; then
+                if [ "${ARCH}" == "s390x" ]; then
```

Comment:
Can you please try with this one liner. 
`mv "${install_dir}"/yq_linux_"${ARCH/x86_64/amd64}" "${install_dir}"/yq`

Reply:
sure, thanks it reduced the code